tools/monster.c

```diff
@@ -2096,12 +1830,13 @@ void monster(uint64_t* to_context) {
   print("(set-option :incremental true)\n");
   print("(set-logic QF_BV)\n\n");
 
-  timeout = max_execution_depth - get_execution_depth(to_context);
+  timeout = 1;
 
   while (1) {
+
     if (debug_merge)
       if (from_context != (uint64_t*) 0)
-        printf4("; switching from context %u to context %u (merge locations: %x, %x)\n",
+        printf4("; switching from context %d to context %d (merge locations: %x, %x)\n",
```

Comment:
Same as above.

Reply:
Overlooked that as well just like before, also changed it to %u.